partcad/tests/unit/test_sketch.py

```
@@ -0,0 +1,35 @@
+# Author: Roman Kuzmenko
+# Created: 2025-01-12
+#
+# Licensed under Apache License, Version 2.0.
+#
+
+import asyncio
+
+import partcad as pc
+
+
+def test_sketch_get_dxf():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_sketch_get_dxf`

Add return type annotation: `None`

(ANN201)